src/calendar/day/period/index.js

```diff
@@ -130,7 +130,7 @@ class Day extends Component {
 
     if (this.props.marking) {
       containerStyle.push({
-        borderRadius: 17
+        borderRadius: this.style.base.borderRadius
```

Comment:
This is the actual fix of this PR, right? Where is the 'style.base.borderRadius' is coming from? Why you're not using the 'theme' object and 'appStyle' to pass styles to the component?

Reply:
If I remember correctly, this is what I wanted to do, but the border radius was hardcoded at the time, i.e. this part of the styling could not be overwritten by the appStyle object.
I apologise of this is obsolete by now, but I don't have access to the companies code anymore, so I'm not 100% sure in what situation I needed this.